integration-tests/tests/workspace-dashboard.js

```diff
@@ -0,0 +1,116 @@
+// This test is owned by the Workspaces Team.
+const _ = require('lodash/fp')
+const { clickNavChildAndLoad, viewWorkspaceDashboard, withWorkspace } = require('../utils/integration-helpers')
+const { assertNavChildNotFound, findText } = require('../utils/integration-utils')
+const { withUserToken } = require('../utils/terra-sa-utils')
+
+
+const testGoogleWorkspace = _.flow(
+  withWorkspace,
+  withUserToken
+)(async ({ page, token, testUrl, workspaceName }) => {
+  await page.goto(testUrl)
+  await viewWorkspaceDashboard(page, token, workspaceName)
+  await findText(page, 'About the workspace')
+  // Click on each of the expected tabs
+  await clickNavChildAndLoad(page, 'data')
+  await clickNavChildAndLoad(page, 'notebooks')
+  await clickNavChildAndLoad(page, 'workflows')
+  await clickNavChildAndLoad(page, 'job history')
+})
+
+const googleWorkspaceDashboard = {
+  name: 'google-workspace',
+  fn: testGoogleWorkspace
+}
+
+const setAjaxMockValues = async (testPage, namespace, name, workspaceDescription) => {
+  const workspaceInfo = {
+    attributes: { description: workspaceDescription },
+    authorizationDomain: [],
+    bucketName: '',
+    createdBy: 'cahrens@gmail.com',
+    createdDate: '2022-04-12T18:12:25.912Z',
+    googleProject: '',
+    isLocked: false,
+    lastModified: '2022-04-12T18:12:26.199Z',
+    name,
+    namespace,
+    workspaceId: '95acc5fe-4c68-4e60-9c2e-c0863af11dee',
+    workspaceVersion: 'v2'
+  }
+  const azureWorkspacesListResult = [{
+    accessLevel: 'OWNER',
+    public: false,
+    workspace: workspaceInfo,
+    workspaceSubmissionStats: { runningSubmissionsCount: 0 }
+  }]
+
+  const azureWorkspaceDetailsResult = {
+    azureContext: {
+      managedResourceGroupId: 'mrg-terra-workspace-20220412104730',
+      subscriptionId: '3efc5bdf-be0e-44e7-b1d7-c08931e3c16c',
+      tenantId: 'fad90753-2022-4456-9b0a-c7e5b934e408'
+    },
+    workspaceSubmissionStats: { runningSubmissionsCount: 0 },
+    accessLevel: 'OWNER',
+    owners: ['cahrens@gmail.com'],
```

Comment:
Nitpick: You may want to use an explicitly fake user + email domain here (i.e, example@example.com) in case anyone scrapes this code.

Reply:
Good point… I will also change the Azure subscription info so that it isn't real. I just dumped the actual content from my workspace to populate the test.

Reply:
Updated and force pushed.